rllib/algorithms/algorithm.py

```diff
@@ -2138,15 +2144,16 @@ def validate_env(env: EnvType, env_context: EnvContext) -> None:
         """
         pass
 
-    def try_recover_from_step_attempt(
-        self, error, worker_set, ignore, recreate
-    ) -> None:
+    def try_recover_from_step_attempt(self, error, worker_set, ignore, recreate) -> int:
         """Try to identify and remove any unhealthy workers (incl. eval workers).
 
         This method is called after an unexpected remote error is encountered
         from a worker during the call to `self.step()`. It issues check requests to
         all current workers and removes any that respond with error. If no healthy
         workers remain, an error is raised.
+
+        Returns:
+            The number of remote workers recovered.
```

Comment:
"recovered" -> "recreated" ?
 recovered sounds like we are reusing the failed one.

Reply:
done. changed it to recreated. sounds good to you? since we always recreate workers.